docs/permission/plugin-authors/03-adding-a-resource-permission-check.md

```diff
@@ -139,6 +142,7 @@ const createTodoListPermissionRule = makeCreatePermissionRule<
 export const isOwner = createTodoListPermissionRule({
   name: 'IS_OWNER',
   description: 'Should allow only if the todo belongs to the user',
+  resourceType: 'todo-item',
```

Comment:
Would it make sense to add a comment pointing to the fact that this should be the same value defined in the constant `TODO_LIST_RESOURCE_TYPE` or are we 100% confident that it is evident?

Reply:
Actually this might be the best middle ground - use the string but call it out. 👍